Makefile

```diff
@@ -0,0 +1,14 @@
+# Building docker image
+VERSION = "v1.0.0"
+IM = shalsh/phenoscape-pipeline-tools
```

Comment:
`IM` is pretty cryptic. I had to read the whole Makefile to understand what it is. Why be cryptic? The point of a Makefile is so that one can quickly grasp what is going on.  Why not call this something like `DOCKER_IMAGE`. I.e., call it what it is, not some cryptic abbreviation of it.